### PR TITLE
Fix EE Menu

### PR DESCRIPTION
**Pull request by felixheidecke**
- Fix base style
- Fix enterprise menu

### CODE REVIEW

**Comment by felixheidecke** [comment]
Fixes https://github.com/owncloud/market/pull/77

File: src/styles/theme.scss [hunk 1]
```diff
@@ -37,4 +37,8 @@
   a:hover {
     text-decoration: none;
   }
+
+  img {
```
Comment (PVince81) [inline src/styles/theme.scss]: assuming this is not global and won't mess up other apps ?

**Comment by PVince81** [comment]
what exactly is fixed ? the EE menu arrow ?

I ran make then copied the contents of "css" and "js" to "owncloud-enterprise/apps/market/" but the arrow is still not there:
ee-missing-arrow

even after clearing the cache.

If you didn't fix that please state what exactly was fixed and don't leave us guessing, thanks

**Comment by felixheidecke** [comment]
Sorry … I posted the wrong Issue. :-(
It fixes: https://github.com/owncloud/market/issues/54

File: src/styles/theme.scss (hunk 1 above)
Comment (felixheidecke) [inline src/styles/theme.scss]: No, it's scoped to market and the #content-wrapper

**Comment by PVince81** [comment]
@felixheidecke wip or ready for review ?

**Comment by felixheidecke** [comment]
Actually … this is outdated since it has been addressed in the regular frontend work outside of this PR.

**Comment by felixheidecke** [comment]
ready for review!

**Comment by felixheidecke** [comment]
Aaaaaand I'm back with my stinky GIT skills 😛

**Comment by PVince81** [comment]
@felixheidecke did you find a tester for this ?

@davitol maybe

**Comment by davitol** [comment]
Market
screen shot 2017-07-11 at 11 15 14

Files

screen shot 2017-07-11 at 11 14 58

The empty space when for example activity is not enabled is still there.

screen shot 2017-07-11 at 11 13 47

But LGTM so @felixheidecke please create a clean PR with these changes and ping me to retest. THX.

**Comment by SergioBertolinSG** [comment]
@felixheidecke you need to add your correct name or email in the first commit 674551fc08d734ea59b070f91fb4583085413018

**Comment by felixheidecke** [comment]
This PR is obsolete and too far behind. New PR will follow.

**Pull request closed without merging**